src/rules/quotemarkRule.ts

```diff
@@ -37,6 +37,10 @@ interface Options {
     avoidTemplate: boolean;
 }
 
+function isQuoteMark(value: QUOTE_MARK | string): value is QUOTE_MARK {
```

Comment:
Nit: you can just put `value: string`, since `QUOTE_MARK` is a subset of `string`.